.changeset/README.md

```diff
@@ -1,8 +1,32 @@
-# Changesets
+# Changeset sjabloon
 
-Hello and welcome! This folder has been automatically generated by `@changesets/cli`, a build tool that works
-with multi-package repos, or single-package repos to help you version and publish your code. You can
-find the full documentation for it [in our repository](https://github.com/changesets/changesets)
+Kopieer en plak het onderstaande sjabloon. Je kunt hiervoor de kopieer knop linksboven in het template gebruiken.
 
-We have a quick list of common questions to get you started engaging with this project in
-[our documentation](https://github.com/changesets/changesets/blob/main/docs/common-questions.md)
+```markdown
+---
+"@organisatie/package-naam": major
+---
+
+- 
+```
+
+Vul tussen de twee sets `---` in voor welke packages je allemaal veranderingen hebt doorgevoerd. Zet elk package tussen
+dubbele aanhalingstekens op een nieuwe regel.
+
+Bijvoorbeeld:
+
+```yaml
+---
+"@nl-design-system-unstable/voorbeeld-design-tokens": minor
+---
+```
+
+Gebruik `major` voor ‘breaking changes’, `minor` voor nieuwe features en `patch` voor bugfixes. Lees de
```

Comment:
Ik denk dat het misschien duidelijker en meer copy-pastbaar is als je een unordered list gebruikt:

- `"@organisatie/package-naam": major` voor foo
- `"@organisatie/package-naam": minor` voor bar
- `"@organisatie/package-naam": patch` voor quux